drivers/wasapi/audio_driver_wasapi.cpp

```diff
@@ -558,9 +558,7 @@ Error AudioDriverWASAPI::init() {
 	target_latency_ms = Engine::get_singleton()->get_audio_output_latency();
 
 	Error err = init_output_device();
-	if (err != OK) {
-		ERR_PRINT("WASAPI: init_output_device error");
-	}
+	ERR_FAIL_COND_V_MSG(err != OK, err, "WASAPI: init_output_device error.");
 
 	exit_thread.clear();
```

Comment:
Unsure if this should be moved up, it is above in the alsa code and could cause threading problems if not fired with this change, it's put at the top in some cases, I don't know what the exact structure should be for that

Reply:
I can see it's above in the output initialisation of the alsa & pulseaudio drivers. The flag is only set in the thread entry point and in `AudioDriverWASAPI::finish`. Not sure about the structure too. Could be moved up for consistency.

Reply:
moved it up